md/what-is-bonita.md

```diff
@@ -1,59 +1,62 @@
 # What is Bonita?
```

Comment:
Now we need to remove this page from the repository. How to check if some links navigate to this page?